install_config/configuring_sdn.adoc

```diff
@@ -183,6 +192,27 @@ networkConfig:
 *redhat/openshift-ovs-networkpolicy* for the *ovs-networkpolicy* plug-in
 ====
 
+[[growing-the-service-network]]
+== Growing the Service Network
+
+If you are running low on addresses in your Service network and can expand the range so that it includes the current range at the beginning of the new range, then you can:
```

Comment:
Maybe:

If you are running low on addresses in your service network, you can expand the range, ensuring that it includes the current range at the beginning of the new range.